src/includes/enriching-events/set-transaction-name/flutter.mdx

```diff
@@ -0,0 +1,3 @@
+```dart
+Sentry.configureScope((scope) { scope.transaction = "UserListView"; });
```

Comment:
If it's the same, it will inherit from the dart platform, this file is unnecessary.